server/tls/tls.js

```diff
@@ -21,6 +21,10 @@ function getCredentials() {
       caPath,
     });
     return createSecure(pk, cert, ca, serverName, verifyHost);
+  } else if (caPath !== undefined && certPath === undefined) {
```

Comment:
as another option we can modify `readCredsFromCertFiles` to return `ca`  even if pk or cert is not provided

Reply:
Sounds good. I've changed the `if` statement in `tls.js` to reflect that.
I've also changed the `if` condition for throwing within `readCredsFromCertFiles`.
Thanks!